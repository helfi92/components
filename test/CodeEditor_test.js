import React from 'react';
import CodeEditor from '../src/components/CodeEditor';

describe('<CodeEditor />', () => {
  it('should render correctly', () => {
    const text = JSON.stringify({ foo: 'bar' });
    const codeEditor = shallow(
      <CodeEditor value={text} />
    );

    expect(codeEditor).toMatchSnapshot();
  });
});
