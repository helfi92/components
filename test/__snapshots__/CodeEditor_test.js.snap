// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CodeEditor /> should render correctly 1`] = `
<CodeEditor
  className={null}
  classes={
    Object {
      "root": "CodeEditor-root-1",
    }
  }
  onChange={null}
  options={null}
  value="{\\"foo\\":\\"bar\\"}"
/>
`;
